clone_all_repos.py

```
@@ -0,0 +1,534 @@
+    """Check if required dependencies are available"""
+    missing = []
+
+    # Check if requests is available
+    try:
+        import requests
+    except ImportError:
+        missing.append("requests")
+
+    # Check if git is available
+    try:
+        subprocess.run(['git', '--version'], capture_output=True, check=True)
```

Ruff (0.13.3)

Starting a process with a partial executable path

(S607)